### DISCUSSION TITLE
DB backup

### DISCUSSION THREAD

**Discussion by luke-on-security** [monarc-project/MonarcAppFO#435] [Q&A] [answered]
Hi,
I installed a MONARC app for several orgs on a server. Without a backoffice MONARC.
Is there a recommended way to backup the db of the orgs ?

**Comment by cedricbonhomme** [comment, marked as answer]
I can recommend you a solution based on mysqldump.
Below is a link to a script we recently tested:
https://gist.github.com/cedricbonhomme/a449c19ea5e69619cccc548d3970abf8

As you can see you have the option to version the backup directory.
Then, the backup should be archived and transfered to another server. It's out of scope here.

**Reply by luke-on-security** [reply]
thx!
